### PR TITLE
Added canDeactivate guard to login, register and reset password pages

### PR DESCRIPTION
**Pull request by DaniilPoiarkov**
Added canDeactivate guards to login, register and reset password pages

### CODE REVIEW

**Comment by Setup1337** [comment]
Should it work when user gave some data?

**Comment by DaniilPoiarkov** [comment]
> Should it work when user gave some data?

If u paste some data in fields and try to go to another page browser will ask you for confirmation

**Comment by Setup1337** [comment]
> 

image
Works even with empty data

File: backend/Tasque.WebAPI/appsettings.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
   },
   "AllowedHosts": "*",
   "ConnectionStrings": {
-    "TasqueDb": "Host=localhost; Database=tasquedb; Username=postgres; Password=mysecretpassword"
```
Comment (ismv) [inline backend/Tasque.WebAPI/appsettings.json]: You sure you need to update this?

**Comment by DaniilPoiarkov** [comment]
> > 
> 
> image Works even with empty data

Fixed

File: backend/Tasque.WebAPI/appsettings.json (hunk 1 above)
Comment (DaniilPoiarkov) [reply backend/Tasque.WebAPI/appsettings.json]: > You sure you need to update this?

Removed this change

**Review by ismv — APPROVED** [review]

**Review by andpii — APPROVED** [review]

**Pull request merged**